### ISSUE TITLE
Wrong placement of an extra comment closing

### ISSUE THREAD

**Issue by BugHunter2k** [jakubpawlowicz/clean-css#965]
### Environment

* clean-css version - `npm ls clean-css`:
clean-css@4.1.7

* node.js version - `node -v`:
v6.6.0

* operating system:
Kubuntu 17.04

### Configuration options
defaults

### Input CSS

```css
[class*='col-'] {
	float: left;
	padding-right: 30px; /* column-space */
}

[class*='col-']:last-child {
    float: left;
/*     padding-right: 0; /* column-space */ */
}


.grid {
	width: 100%;
	max-width: 1200px;
	min-width: 755px;
	margin: 0 auto;
}
```

### Actual output CSS

```css
[class*=col-]{float:left;padding-right:30px}[class*=col-]:last-child{float:left}*/ .grid{width:100%;max-width:1200px;min-width:755px;margin:0 auto}
```

### Expected output CSS

```css
[class*=col-]{float:left;padding-right:30px}[class*=col-]:last-child{float:left}.grid{width:100%;max-width:1200px;min-width:755px;margin:0 auto}
```

**Comment by jakubpawlowicz** [comment]
Please correct me if I'm wrong but clean-css should not remove the `*/ ` piece, consider the following example: https://jsfiddle.net/z4vxqafd/.

The first `*/` closes both comments and the parsers treat the second `*/` as a part of next property (in given case `background-color` rendering it invalid.

It's a separate question why clean-css puts remaining `*/ ` outside of a rule not inside.

**Comment by BugHunter2k** [comment]
Yes, you are right. The css specs (I haven't checked them before) so my css was faulty.
https://drafts.csswg.org/css-syntax-3/#comment-diagram

But you are right about the placement of the `*/` that  should be inside the rule.

**Comment by jakubpawlowicz** [comment]
It's fixed in 4.1.8, thanks!
